shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -730,6 +735,10 @@ public int getCapacityBytes() {
   public int pin(TachyonURI path) {
     try {
       TachyonFile fd = mTfs.open(path);
+      if (fd == null) {
```

Comment:
What about abstracting this check into a function, since it's used multiple times?